### ISSUE TITLE
Position of translated affidavit

### ISSUE THREAD

**Issue by JuLateX** [tudace/tuda_latex_templates#300]
When adding a translated affidavit to a PhD thesis using the mechanism:
```
\begin{affidavit*}[Babel-Sprachoption]{Überschrift}
Text
\end{affidavit*}
```
it adds the translated affidavit on a new page (I think the `\affidavit` printing the affidavit in German must be responsible for it).

Is it possible to have the translation directly below the German version? It would make more sense sonce it is the same thing.

**Comment by TeXhackse** [comment]
There is `\affidavit`, which is indeed adding a `\clearpage`. We added `\affidavit*` to allow users to disable that. This variant is available since TUDa-CI v3.06.

**Comment by JuLateX** [comment]
It works perfectly, thanks! (Sorry, it is sometimes difficult to understand the documenattion in German.)
